src/Umbraco.Cms.Api.Management/ViewModels/Security/UnLinkLoginRequestModel.cs

```diff
@@ -0,0 +1,15 @@
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+
+namespace Umbraco.Cms.Api.Management.ViewModels.Security;
+
+public class UnLinkLoginRequestModel
+{
+    [Required]
+    [DataMember(Name = "loginProvider", IsRequired = true)]
```

Comment:
Why do we have data member attribute on these?

Reply:
Sorry, bad copy paste